src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(number) {
```

Comment:
Checklist point 2: you don't need to name functions if you return them right away.
```suggestion
  return function (number) {
```